cit3.0-web/src/components/OpportunityView/Resource.js

```diff
@@ -56,6 +56,10 @@ function displayResources(resources) {
         : "No";
     }
 
+    // if (resource[0] === "Nearest Lake") {
+    //   resource[1].
+    // }
+
```

Comment:
I think this is the only change needed. Can we remove this?

Reply:
You betcha!  I literally knew as I commented out that I'd forget this!  Sigh.